ps5Controller/ps5Controller/HomeModule/View/HomeCollectionViewCell.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+final class HomeCollectionViewCell: UICollectionViewCell {
+    static let identifier = "HomeCollectionViewCell"
+    static func nib() -> UINib {
```

Comment:
computed var?, identifier class ismiyle aynı, bunu kolay'dan tüm cell'lere implemente edebileceğin bi extension yazabilir misin?